steampy/client.py

```diff
@@ -241,7 +241,69 @@ def make_offer(self, items_from_me: List[Asset], items_from_them: List[Asset], p
         if response.get('needs_mobile_confirmation'):
             return self._confirm_transaction(response['tradeofferid'])
         return response
+        
+    @login_required
+    def get_escrow_duration(self, trade_offer_url: str) -> int:
+        start = trade_offer_url.index('steamcommunity.com') + len('steamcommunity.com')
+        end_trade_url=trade_offer_url[start:]
+
+        headers = {'Referer': self.COMMUNITY_URL + end_trade_url,
```

Comment:
extract 247-248 lines to some method called get_escrow_duration_referer, because lines 247-250 only serve for seting referer in headers.

I like 1 layer of abstraction per method